integration-tests/src/main/resources/crafter/profile/extension/init-data.js

```diff
@@ -92,7 +92,8 @@ if (db.tenant.count() == 0) {
                     }
                 ]
             }
-        ]
+        ],
+        "cleanseAttributes": false
```

Comment:
What is this for?

Reply:
If set to true (the default) the service will escape the value of the attributes and was causing the test to fail